src/ipc/dbus/DBusObjectRegistry.h

```diff
@@ -76,16 +74,22 @@ class FaceliftIPCLibDBus_EXPORT DBusObjectRegistry : public QObject
 
     void unregisterObject(const QString &objectPath);
 
-    const Content &objects(bool blocking);
+    QMap<QString, QString> objects(bool blocking);
```

Comment:
Because it is shallow copy. And if you see at the implementation, there is one path that returns directly from proxy object (this would cause returning reference to local variable)